src/main/java/com/codeborne/selenide/impl/HttpHelper.java

```diff
@@ -22,7 +22,7 @@
 public class HttpHelper {
 
   private static final Pattern FILENAME_IN_CONTENT_DISPOSITION_HEADER =
-    Pattern.compile(".*filename\\*?=\"?((.+)'')?([^\";?]*)\"?(;charset=(.*))?.*", CASE_INSENSITIVE);
+    Pattern.compile(".*filename\\*? *= *\"?((.+)'')?([^\";?]*)\"?(;charset=(.*))?.*", CASE_INSENSITIVE);
```

Comment:
Please use `\\s` instead of just ` ` - it means "any space symbol" in regexp.

Reply:
Looks like it could be dangerous to use it in this way, because `enter` is also could be in `/s` in the regex

Reply:
Agree. No changes needed.